watson_developer_cloud/speech_to_text_v1.py

```diff
@@ -41,6 +41,8 @@ def recognize(self, audio, content_type, continuous=None, model=None,
         Returns the recognized text from the audio input
         """
         headers = {'content-type': content_type}
+        if continuous is None:
```

Comment:
I think this may be due to changes in the underlying repo, but the changes to speech_to_text should be omitted -- these lines were removed to fix a deprecation bug.